source/libnormaliz/cone.cpp

```diff
@@ -916,13 +919,16 @@ void Cone<Integer>::process_multi_input_inner(map< InputType, vector< vector<Int
             case Type::normalization:
             case Type::cone_and_lattice:
                 nr_cone_gen++;
+                nr_latt_gen++;
+                break;
```

Comment:
This preserves semantics by making explicit what happens; but please verify that this is actually the intended behavior.